### ISSUE TITLE
AutoPairs should be off in incremential search 

### ISSUE THREAD

**Issue by ericllazarus** [myTerminal/super-emacs#9]
Not sure I love https://www.emacswiki.org/emacs/AutoPairs or whatever super emacs uses but for sure I don't want it on when I'm in the search buffer. Make sense?

**Comment by myTerminal** [comment]
Yes. I've been facing this issue as well since I've started using autopairs, but I never bothered to disable it for certain scenarios. This is a good idea, I'll investigate...

**Comment by myTerminal** [comment]
I recall now that I replaced it with [electric-pair-mode](https://github.com/myTerminal/super-emacs/blob/eeab8a11208542d95e5304ca01c92336d55e165d/.emacs.d/super-emacs/config-misc.el#L22) long back. I also realize that I don't face that issue anymore.

**Comment by myTerminal** [comment]
@ericllazarus, do we still have this issue? If not, can we close this issue?

**Comment by ericllazarus** [comment]
Let me check and get back to you.

**Comment by ericllazarus** [comment]
When I search (control-S) I don't want this mode.
When I am writing a keyboard macro, I don't want it
In the minibuffer in general, I suppose I don't want it like control-x
control-f find file.

Unrelated, when I kill to end of line control-K, I don't want to kill the
next line.

Thanks!

Eric
